grpc/src/test/java/com/linecorp/armeria/server/grpc/GrpcServiceServerTest.java

```diff
@@ -1228,6 +1248,23 @@ void grpcContext(UnitTestServiceBlockingStub blockingClient) throws Exception {
         requestLogQueue.take();
     }
 
+    @Test
+    void autoCompression() throws Exception {
```

Comment:
- This class is already huge. Should move this into `GrpcServiceAutoCompressTest`?
- Should we use Armeria gRPC client for testing?